app/containers/MessageBox/index.js

```diff
@@ -927,7 +927,7 @@ class MessageBox extends Component {
 				/>
 				<TextInput
 					ref={component => this.component = component}
-					style={styles.textBoxInput}
+					style={{ ...styles.textBoxInput, color: themes[theme].bodyText }}
```

Comment:
Use array instead of object.

```suggestion
					style={[styles.textBoxInput, { color: themes[theme].bodyText }]}
```

Reply:
Done,